velodyne_pointcloud/include/velodyne_pointcloud/rawdata.hpp

```diff
@@ -135,15 +176,20 @@ class RawData final
 
 private:
   /** configuration parameters */
-  struct Config
+  typedef struct
   {
     std::string model;
-    double min_range;             ///< minimum range to publish
+    std::string calibrationFile;  ///< calibration file name
```

Comment:
Why change the ordering of this struct? And why is it that the path to the calibration file is needed here now when it was not before?

Reply:
Only reason is that the ROS1 fork had that included here. I will double check if its used and if not remove here.